src/components/study-container.js

```diff
@@ -304,6 +306,35 @@ export function StudyContainer({ view, onChangeTab }) {
         }
     }, [studyUuid, loadTree]);
 
+    useEffect(() => {
+        const appName = 'GridStudy';
```

Comment:
get the original title from the html <title>

Reply:
fixed by setting the initial title in the component state